.flutter-plugins

```diff
@@ -0,0 +1,5 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
This file should not be tracked. Needs to be removed from the cache (`git rm --cached .flutter-plugin-dependencies`) and added to the gitignore.